docs/v2.4/releaseNotes.yml

```diff
@@ -78,6 +78,14 @@ items:
           shutdown or pod restart to fix. We've added some fixes
           to prevent the traffic-manager from getting into this state.
 
+      - type: bugfix
+        title: The traffic agent will backoff, retry after a failed connection to Ambassador Cloud
```

Comment:
I would change this title to read something like: 
Improved proprietary agent connectivity